optuna/study.py

```diff
@@ -520,7 +522,8 @@ def _run_trial(self, func, catch):
             # environments (e.g., services that use computing containers such as CircleCI).
             # Please refer to the following PR for further details:
             # https://github.com/pfnet/optuna/pull/325.
```

Comment:
How about exposing similar information to the end user (this comment aimed toward us maintainers can then be removed) to demonstrate the practical usage of this argument?

Reply:
Thank you for pointing it out. That sounds good.
I'll add such an explanation to the document of `force_garbage_collection` argument.